MapboxNavigation/NavigationMapView.swift

```diff
@@ -1335,3 +1353,96 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         enableFrameByFrameCourseViewTracking(for: 3)
     }
 }
+
+// MARK: - Building Extrusion Highlighting
+
+extension NavigationMapView {
+       
+    /**
+     Receives coordinates for searching the map for buildings. If buildings are found, they will be highlighted in 2d or 3d depending on the `in3D` value.
+     `in3D` defaults to true
+     `buildingHighlightingEnabled` must be set to true before this function will work.
```

Comment:
Make this a `- precondition:` command. I think the precondition is actually that the setting should be set to `true` well before this method is called; otherwise, it would be convenient for us to simply set it to true at the beginning of this method (which wouldn’t be such a bad idea anyways).

Reply:
This comment is actually no longer accurate, I've added ability to control building highlight via `DestinationBuildingHighlightType` enum. On the other hand user can perform building highlight via `NavigationMapView` independently.